pineko.cli.toml

```diff
@@ -0,0 +1,15 @@
+[general]
+nnpdf=true
+
+[paths]
+# inputs
+grids = "./theory_productions/data/grids"
+operator_card_template_name = "_template.cli.toml"
```

Comment:
I thought the templaets were .yaml file? Has this changed in eko?

Reply:
they are yaml files

Reply:
Fixed in [5e89cb9](https://github.com/NNPDF/pineko/pull/215/commits/5e89cb9964aefbae2cef1a9457a09c60b32bb99d).